Add SConscript.local to pass through EB environment to the compiler

author: Jakub Zárybnický (Inuits) 
--- libobjcryst-2017.2.3/sconscript.local	1970-01-01 01:00:00.000000000 +0100
+++ libobjcryst-2017.2.3/sconscript.local	2020-09-22 15:33:59.098143895 +0200
@@ -0,0 +1,10 @@
+Import('env')
+
+import os
+
+env.Replace(CC=os.environ['CC'])
+env.Replace(CXX=os.environ['CXX'])
+env.Replace(CFLAGS=os.environ['CFLAGS'])
+env.Replace(CPPFLAGS=os.environ['CPPFLAGS'])
+env.Replace(CXXFLAGS=os.environ['CXXFLAGS'])
+env.Replace(LDFLAGS=os.environ['LDFLAGS'])

--- libobjcryst-2017.2.3/SConstruct	1970-01-01 01:00:00.000000000 +0100
+++ libobjcryst-2017.2.3/SConstruct	2020-09-22 15:33:59.098143895 +0200
@@ -1,1 +1,1 @@
-    MACOSX_DEPLOYMENT_TARGET
+    MACOSX_DEPLOYMENT_TARGET INTEL_LICENSE_FILE

